src/commands/dev/gcs/headers.rs

```diff
@@ -60,7 +60,7 @@ fn strip_response_headers_prefix(parts: &mut ResponseParts) -> Result<(), failur
         if name.starts_with(HEADER_PREFIX) {
             let header_name = &name[HEADER_PREFIX.len()..];
             let header_name = HeaderName::from_bytes(header_name.as_bytes())?;
-            headers.insert(header_name, value.clone());
+            headers.append(header_name, value.clone());
```

Comment:
should we  write a unit test for this?

Reply:
should probably write unit tests for dev in general

Reply:
but - yes

Reply:
sure; this  one  seems simple  enough to get the ball rolling tho